client/components/FormattingBarLegacy/Controls/controls.scss

```diff
@@ -40,7 +40,7 @@
 		width: 100%;
 		// height: 50px;
 		height: 45px;
-		overflow-y: scroll;
+		overflow-y: auto;
```

Comment:
![image](https://user-images.githubusercontent.com/2208769/88959633-4db37c80-d270-11ea-9add-4cfdd67c30cb.png)

This lives in an obscure part of the Pages editor and really needs to be replaced with the modern formatting bar.